drivers/can/Kconfig.sam

```diff
@@ -0,0 +1,22 @@
+# SAM CAN configuration options
+# Copyright (c) 2021 Alexander Wachter
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_ATMEL_SAM_CAN := atmel,sam-can
+
+config CAN_SAM
+	bool "Atmel SAM CAN driver"
+	default $(dt_compat_enabled,$(DT_COMPAT_ATMEL_SAM_CAN))
+	select CAN_MCAN
```

Comment:
Shouldn't CAN_SAM have a depends on CAN too?

Reply:
No, it's already wrapped in `if CAN` in the higher level Kconfig file.